app/backend/src/couchers/config.py

```diff
@@ -45,6 +45,9 @@
     ("BUG_TOOL_GITHUB_REPO", str),
     ("BUG_TOOL_GITHUB_USERNAME", str),
     ("BUG_TOOL_GITHUB_TOKEN", str),
+    # Sentry
+    ("SENTRY_ENABLED", bool, "0"),
+    ("SENTRY_URL", str, ""),
```

Comment:
Better to not have default in here and just add it to the `backend.*.env`s like others?

Reply:
I think having defaults is nicer, as you prevent code from breaking if we miss this value in one of the .env files. In this case the default is also sensible semantically, since if nothing is configured, it should just not use Sentry.

Reply:
The original thinking was to have basically all config options mandatory: that way you are forced to make sure they're all set correctly, config files will have the whole thing there, and the app will just fail on startup if everything isn't set up correctly. There are two main exceptions to this: `VERSION` and `IN_TEST` because they have special handling.

I guess we could move to having sensible defaults for most config options, but I think it'd be better to stick to one convention and stay consistent.

Reply:
cool, changed it. But beware: not setting these prometheus variables in the environment will crash the app now.

Reply:
There's some env vars in the test config too btw, maybe these should be there too